test/test_example_notebooks.py

```diff
@@ -23,8 +23,11 @@ def test_example_notebooks():
     if len(notebooks_list) == 0:
         raise Exception("No notebooks found!")
 
+    no_test_list = [f"{git_home}/examples/load_waveform_demo.ipynb"]
```

Comment:
For the sake of future Arif, always explain why you are doing non-obvious things like this.

Reply:
Done